content/docs/start/index.md

```diff
@@ -132,18 +138,7 @@ These are captured in our [example-dvc-experiments] repo (see its
 
 </admon>
 
-## Following the Get Started
+## Where To Go Next
 
-Each page in the trails above is more or less independent, especially if you're
-only reading them to get a general idea of the features in question. For better
-learning, try each step yourself from the beginning of any trail. Some of the
-preparation steps may be inside collapsed sections you can click on to expand:
-
-<details>
-
-### Click for an example!
-
-Click the header again to collapse this message. Or move on by picking a page
-from the list above, left-side navigation, or just click `NEXT` below!
-
-</details>
+Pick a page from the list above, the left-side navigation bar, or just click
```

Comment:
This was here before in some other form as you can see. I thought about removing it entirely, but I guess having some navigation-helper text in SOME place is helpful, so we have it in 1 place, and what other place is more suitable then in the first page a new user sees.
Maybe it's ok to keep since, like I mentioned in another comment, keep in mind users might not notice the next/prev buttons 😄

Reply:
I think it's related to the discussion on this page content above. I think the page is way toooooo long. Also We already have a lot of navigation help there. I think two links with some descriptions to the top pages of trails should be enough to serve the "navigational" purpose.

Reply:
@shcheklein  - If you feel strongly about this let me and I'll remove

Reply:
> Oh, that's a different https://github.com/iterative/dvc.org/pull/4460#discussion_r1162708147 then, let's try to separate from this single sentence please.

they are related to my mind, since this sentence serves a navigational purpose _and_ the whole content on the page does the same I think.

> This content was here in this form before:

Yep, it was more a less educational stub about the expandable section. So, it's a completely different meaning to me now. And in the new (very different form and for a different reason) it feels excessive to me.

> Do you still want this completely removed?

your call, I shared my opinion on this (and on the whole page), it's not a blocker :)

> split it - chapter navigation help should come BEFORE the chapter links

It depends on the page length. If everything fits into one screen ( no scroll) - it doesn't matter much where it stays. I think initially the page was way smaller (before we started introducing trails).

Reply:
If it's ok with you then, let's keep it for now.
I will do a followup PR as I mentioned to try and condense more content and maybe unify pages in the trails and this will effect this as well
As per fitting in one screen or not - I think order has to be correct regardless personally. people have different screen sizes, orientation, may read this on mobile / tablet and resize, so we shouldn't count on it too much

Reply:
sounds good!

>  condense more content and maybe unify pages in the trails and this will effect this as well

My 2cs.

I don't feel it's a top priority tbh. If we decide to do this I would think about unifying and making it a single trails again (which has certain challenges).

Things like - let's remove a bit of admons / expandables and or move content into one page w/o deeply changing it - my personal take - they won't move the needle. 

We went from a single page (Quick Start) -> to 3-4 pages (no experiments, no metrics yet, no import/get) -> 7-8 pages with import/get, metrics -> to 3-4 that combine all that stuff -> to now trails since we want to make experiments a bit more of first-class citizen. In my experience - I didn't see much difference so far in all of those. 

It well might be that now we have more ppl on the website, etc (environment is different). 

Anyways, just saying - 2 trails with multiple pages most likely is the same a single trail and most likely is more or less the same as 2 tails with a single page is. The meaningful change is to make it like MLEM, but in case of DVC I don't think it's possible tbh. They are very different products, with different learning curve, etc, etc.